modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/item/selector/FDSViewItemSelector.tsx

```diff
@@ -21,6 +21,11 @@ import {API_URL, FDS_DEFAULT_PROPS} from '../../js/Constants';
 
 import './FDSViewItemSelector.scss';
 
+interface ISelectedItem {
+	externalReferenceCode: string;
+	id: string;
+}
+
```

Comment:
Ideal solution would be to better define the `items` prop, currently it's `any[]` :disappointed: But, we can do this in a separate task. This is more than ok for now.